docker/vnc/Dockerfile

```diff
@@ -17,7 +17,10 @@ RUN apt-get update \
         xfce4-settings \
         xorg \
         xubuntu-icon-theme \
-        xterm \
+        gnome-terminal \
+        ttf-dejavu \
+        git-gui \
```

Comment:
we may want to add `gitk` here otherwise the visualization of the git history doesn't work

Reply:
good point

Reply:
well I've never heard of or used `gitk` but it's an option in the git gui to visualize the history so this pretty much guarantees that a user will ask about it :)